samples/servicecontrol/adapter-rabbitmq-different-topologies/SCTransportAdapter_1/Shipping/Program.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Threading.Tasks;
+using NServiceBus;
+
+class Program
+{
+    static void Main()
+    {
+        AsyncMain().GetAwaiter().GetResult();
+    }
+
+    static async Task AsyncMain()
+    {
+        Console.Title = "Samples.ServiceControl.RabbitMQAdapter.Shipping";
+
+        #region ShippingConfiguration
+
+        var endpointConfiguration = new EndpointConfiguration("Samples.ServiceControl.RabbitMQAdapter.Shipping");
+
+        var transport = endpointConfiguration.UseTransport<RabbitMQTransport>();
+        transport.ConnectionString("host=localhost");
+        transport.UseDirectRoutingTopology();
+
+        endpointConfiguration.SendFailedMessagesTo("adapter_error");
+        endpointConfiguration.AuditProcessedMessagesTo("adapter_audit");
+        endpointConfiguration.HeartbeatPlugin("adapter_Particular.ServiceControl");
```

Comment:
Does the adapter do something special with the hearbeat messages when it forwards them? If not, this is could possibly be configured to just send directly to SC. Endpoints using the direct topology can send to endpoints using the conventional topology, but the opposite isn't true.

Doing that might make things more difficult to understand though, since it complicates the idea of using the adapter to bridge things.